components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -443,5 +446,11 @@
       <span>List</span>
     </label>
   </div>
+  <div>
+    <label>
+      <strong>Capacity</strong>
+      <input type="number" min={1} bind:value={manifestProperties.capacity} />
```

Comment:
just curious why 1 is in curly braces here `min={1}`? Should it also have a value of 1 as well? Just for consistency

Reply:
I prefer using curly braces in svelte files to make sure the type of what I'm passing in. Sometimes I found that `attr="1"` results in a string. 

I'm `bind:value` here so can't add `value`